arangodb/content.md

```diff
@@ -20,36 +18,29 @@ Joins and Transactions are key features for flexible, secure data designs, widel
 
 Furthermore, ArangoDB offers a microservice framework called [Foxx](https://www.arangodb.com/foxx) to build your own Rest API with a few lines of code.
 
-ArangoDB Documentation
-
--	[ArangoDB Documentation](https://www.arangodb.com/documentation)
--	[ArangoDB Tutorials](https://www.arangodb.com/tutorials)
+ArangoDB Documentation - [ArangoDB Documentation](https://www.arangodb.com/documentation) - [ArangoDB Tutorials](https://www.arangodb.com/tutorials)
```

Comment:
I assume that you want to keep the bullet points, so I think this is an unwanted change.